DateUtils.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Module voor het werken met datums.
+ * @module DateUtils
+ */
+define([
+  'dojo/date/stamp',
+  'dojo/date/locale'
+], function (
+  stamp,
+  locale
+) {
+  return {
+    /**
+     * Zet een datum string om in een datum object met tijdsaanduiding.
+     * @param {string} date Datum string in ISO formaat
+     * @returns {Date|String} Het omgezette datum object
+     */
+    convertIsoStringToDateTime: function (date) {
+      if (!date) {
+        return ' - ';
+      }
+      var d = stamp.fromISOString(date);
+      if (d) {
+        return locale.format(d, {
+          datePattern: 'dd-MM-yyyy',
+          timePattern: '\'om \'HH:mm',
+          locale: 'be'
+        });
+      }
+      else
+      {
+        return ' - ';
```

Comment:
ja, moet in apps afgehandeld worden, is ui logica die niet in de util thuishoort imo